Resources/doc/index.rst

```diff
@@ -251,8 +251,9 @@ Here is an example on how to inject the service container into your migrations:
         services:
             Doctrine\Migrations\Version\DbalMigrationFactory: ~
             App\Migrations\Factory\MigrationFactoryDecorator:
-                decorates: Doctrine\Migrations\Version\DbalMigrationFactory
```

Comment:
This is broken, as you don't decorate the service anymore, and so things won't work.

Reply:
No, it is not broken. You need register `Doctrine\Migrations\Version\DbalMigrationFactory` as a service in your service configuration explicitly. There is no autoconfigure on the interface and you need define the migration factory implementation explicitly in your doctrine migrations config (see at line 248).